lib/good_migrations/logic.rb

```diff
@@ -1,5 +1,35 @@
 module GoodMigrations
-  class PreventsAppLoad
+  class Logic
```

Comment:
What's your reasoning for this file name change? Seems like it loses any communication of what the class does

Reply:
My thoughts on the name "PreventsAppLoad":
* it's ambiguous, is `App` the `app/` folder, or application code in general? 
* it's super specific, anything even slightly generic doesn't feel at its place.

So to me it feels like it needs a different name.

My reasoning is that this becomes where the main logic of the gem is. I can name it something else.